scss/bitstyles/settings/shadows.stories.mdx

```diff
@@ -0,0 +1,145 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
+
+<Meta title="Design Tokens/Shadows" />
+
+# Shadows
+
+Bitstyles provides a global list of shadow specifications, to maintain consistency of shadows throughout your components. The shadows are specified in a sass list so they can be translated to various formats as needed — the order of parameters for CSS `box-shadow` is different to the CSS filter `drop-shadow()`. We need to be able to translate to either of those formats.
+
+The default shadows used throughout bitstyles are multi-layered to create a more natural or realistic appearance than you get from a single box-shadow or filter. We provide a generator function that you can use to produce “natural” multi-layered shadows in the correct format for our Sass design system, and functions to translate those to useful formats.
+
+## Generating shadows
+
+The simplest way to override the global shadow design tokens in `$bitstyles-shadows-shadows` is to use `shadow.generate()`:
+
+```scss
+@use '../tools/shadow';
+
+$bitstyles-shadows-shadows: (
+  'default': shadow.generate(
+    $color: rgba(palette.get('brand-1'), 0.15),
+    $offset: 2rem,
+    $blur-radius: 2rem,
+  ),
+);
+```
+
+This creates a specification for a multi-layered shadow using the `brand-1` color with a `2rem` blur, offset `2rem` downwards, which looks like the following examples.
+
+<Canvas>
+  <Story id="utilities-shadow--u-shadow-default" />
+  <Story id="utilities-drop-shadow--u-drop-shadow-default" />
+</Canvas>
+
+### Parameters
+
+|Name|Format|Description|
+|--|--|--|
+|$color|color|The color to be used. Probably you want a low-opacity version of a color. Avoid using pure black unless your design calls for it, as that can look flat, especially on non-white backgrounds.|
+|$offset|number|Vertical offset for the shadow. Include the units unless it’s zero. Optional, defaults to `0`.
+|$blur-radius|number|Maximum distance to blur the shadow. The extra layers in this multi-layer shadow will be smaller than the value you pass. Include the units.|
+|$layers|number|Number of layers of in the shadow. Optional, defaults to `4`.|
```

Comment:
Oh! This explains a lot. Maybe some of my earlier comments don't make sense any more. Just ignore them.

Reply:
Ok! :D Can docs on this be made clearer maybe? Or is this a problem because of the order the files are presented in github?